### ISSUE TITLE
Desconexion repentina del servidor

### ISSUE THREAD

**Issue by njllanos** [IIC2233/Syllabus#579]
### Prerrequisitos
(Marcar colocando una `X` entre los corchetes los ítems que ya hiciste, así: `"[X]"`)

* [X] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/3.-Reglas-del-foro-(issues)).
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/Syllabus/issues/).
* [X] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía ([click aquí para ir al compilado](/IIC2233/Syllabus/issues/504)).
* [X] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello ([click aquí para ir a esa _issue_](/IIC2233/Syllabus/issues/503)).
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al ayudante de Bienestar o al correo del curso (coordinacion.iic2233@gmail.com).
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla.
* [X] De ser necesario, colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado una falta a la ética.

### Duda

Hola en el enunciado aprece en cuanto a la desconexion repentina que si "servidor quien se desconecta, cada cliente conectado debe mostrar un mensaje en la ventana (ya sea como texto plano o pop-up) explicando la situación, antes de cerrar el programa". Esto significa que si al iniciar el cliente este no puede conectarse al servidor el programa se cierra instantanteamente, o solo contempla la desconexion luego de haberse previamente conectado (dado que es "repentina")?

**Comment by dlv237** [comment]
Efectivamente, solo contempla la desconexión del servidor cuando el cliente esta conectado. 

En el enunciado se menciona que primero debe iniciarse el servidor y seguido a esto el cliente. Si el servidor no está andando, el cliente debe cerrarse instantáneamente.
